sklearn/cluster/_hierarchical_fast.pyx

```diff
@@ -451,7 +451,7 @@ def single_linkage_label(L):
     return _single_linkage_label(L)
 
 
-# Implements MST-LINKAGE-CORE from https://arxiv.org/abs/1109.2378
+# Implements MST-LINKAGE-CORE from :arxiv:`<1109.2378>`
```

Comment:
This link is commented out, then not rendered in the documentation (see my [previous comment](https://github.com/scikit-learn/scikit-learn/pull/21418#issuecomment-984037177)).
I believe it is better to revert this change, otherwise the link is not copy-pastable.